apps/meteor/app/custom-sounds/server/startup/custom-sounds.js

```diff
@@ -46,6 +46,9 @@ Meteor.startup(function () {
 
 		const file = await RocketChatFileCustomSoundsInstance.getFileWithReadStream(fileId);
 		if (!file) {
+			res.writeHead(404);
```

Comment:
@sampaiodiego I've changed this to a 404 error because it was doing nothing here, leading to a timeout error. I don't think this should be considered a breaking change.

Reply:
that's great.. we desperately need tests these endpoints =)